BankSDK/GiniBankSDKExample/Tests/SettingsViewControllerTests.swift

```diff
@@ -7,86 +7,264 @@
+				return
+			}
+			data.isActive = true
+			configuration.onlyQRCodeScanningEnabled = data.isActive
+			
+			XCTAssertTrue(configuration.onlyQRCodeScanningEnabled,
+						  "qr code scanning should be enabled in the gini configuration")
+		}
+	}
+	
+	func testQrCodeScanningOnlySwitchOff() {
+		if case .switchOption(var data) = sectionData[2] {
+			guard data.type == .qrCodeScanningOnly else {
+				XCTFail("Expected type `qrCodeScanning`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = false
+			configuration.onlyQRCodeScanningEnabled = data.isActive
+			
+			XCTAssertFalse(configuration.onlyQRCodeScanningEnabled,
+						   "qr code scanning should not be enabled in the gini configuration")
+		}
+	}
+	
+	func testMultipageSwitchOn() {
+		if case .switchOption(var data) = sectionData[3] {
+			guard data.type == .multipage else {
+				XCTFail("Expected type `multipage`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = true
+			configuration.multipageEnabled = data.isActive
+			
+			XCTAssertTrue(configuration.multipageEnabled,
+						  "multipage should be enabled in the gini configuration")
+		}
+	}
+
+	func testMultipageSwitchOff() {
+		if case .switchOption(var data) = sectionData[3] {
+			guard data.type == .multipage else {
+				XCTFail("Expected type `multipage`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = false
+			configuration.multipageEnabled = data.isActive
+			
+			XCTAssertFalse(configuration.multipageEnabled,
+						   "multipage should not be enabled in the gini configuration")
+		}
+	}
+	
+	func testFlashToggleSwitchOn() {
+		if case .switchOption(var data) = sectionData[4] {
+			guard data.type == .flashToggle else {
+				XCTFail("Expected type `multipage`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = true
+			configuration.flashToggleEnabled = data.isActive
+			
+			XCTAssertTrue(configuration.flashToggleEnabled,
+						  "flashToggle should be enabled in the gini configuration")
+		}
+	}
+	
+	func testFlashToggleSwitchOff() {
+		if case .switchOption(var data) = sectionData[4] {
+			guard data.type == .flashToggle else {
+				XCTFail("Expected type `multipage`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = false
+			configuration.flashToggleEnabled = data.isActive
+			
+			XCTAssertFalse(configuration.flashToggleEnabled,
+						   "flashToggle should not be enabled in the gini configuration")
+		}
+	}
+	
+	func testbottomNaviagtionBarSwitchOn() {
+		if case .switchOption(var data) = sectionData[5] {
+			guard data.type == .bottomNaviagtionBar else {
+				XCTFail("Expected type `bottomNaviagtionBar`, found a different one: \(data.type)")
+				return
+			}
+			data.isActive = true
+			configuration.bottomNavigationBarEnabled = data.isActive
+			
+			XCTAssertTrue(configuration.bottomNavigationBarEnabled,
+						  "flashToggle should be enabled in the gini configuration")
```

Comment:
[Clarification]: I think it should be `bottomNavigationBarEnabled` not `flashToggle` :)?